domain/src/main/java/no/difi/meldingsutveksling/domain/webhooks/Subscription.java

```diff
@@ -0,0 +1,88 @@
+package no.difi.meldingsutveksling.domain.webhooks;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import no.difi.meldingsutveksling.nextmove.AbstractEntity;
+import no.difi.meldingsutveksling.validation.OneOf;
+import no.difi.meldingsutveksling.validation.WebhookFilter;
+import no.difi.meldingsutveksling.validation.group.ValidationGroups;
+import org.hibernate.annotations.OptimisticLockType;
+import org.hibernate.annotations.OptimisticLocking;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Null;
+
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(name = "webhook_subscription")
+@ApiModel(description = "Webhook subscription")
+@OptimisticLocking(type = OptimisticLockType.NONE)
+public class Subscription extends AbstractEntity<Long> {
+
+    @NotNull(groups = ValidationGroups.Create.class)
+    @ApiModelProperty(
+            position = 1,
+            value = "A label to remember why it was created. Use it for whatever purpose you'd like.",
+            required = true,
+            example = "Outgoing DPO messages"
+    )
+    private String name;
+
+    @Null(groups = ValidationGroups.Create.class)
+    @Override
+    @JsonProperty
+    @ApiModelProperty(
+            position = 2,
+            value = "Id",
+            example = "1")
+    public Long getId() {
+        return super.getId();
+    }
+
+    @NotNull(groups = ValidationGroups.Create.class)
+    @ApiModelProperty(
+            position = 3,
+            value = "URL to push the webhook messages too",
+            required = true,
+            example = "https://webhook.site/bb8bb41a-c559-47ed-84f1-1846ece5d590"
+    )
+    private String pushEndpoint;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1213)